openpathsampling/random.py

```diff
@@ -0,0 +1,25 @@
+from __future__ import absolute_import
```

Comment:
This is needed as `python 2.7` does not understand that I don't want to import this file whenever I say `import random` in this file...
```python
Python 2.7.15 | packaged by conda-forge | (default, Mar  5 2020, 14:56:06) 
[GCC 7.3.0] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> from openpathsampling.random import random
>>> random.random
<module 'openpathsampling.random' from 'openpathsampling/random.py'>
>>> random.random.random
<module 'openpathsampling.random' from 'openpathsampling/random.py'>
>>> random.random.random.random
<module 'openpathsampling.random' from 'openpathsampling/random.py'>
```